data/sea/54-zebra.h

```diff
@@ -155,7 +156,7 @@ static int64_t zebra_translate_column
                 istring_t *strings = anemone_mempool_alloc (mempool, count * sizeof (istring_t));
                 int offset = 1;
                 for (int i = 0; i != count; ++i) {
-                    int64_t index = i + elem_start;
+                    int64_t index = i;
```

Comment:
I'm not sure about this. I *think* this should include the `elem_start`, but that `elem_start` should not have added `+= len` below, and also the recursive call should use `start` instead of `elem_start`.

the `elem_start` refers to the current level - that is the indices/lengths array. then `start` refers to the nested values, so you need to use it when calling `zebra_translate_table`

I think the current code *would* pass as long as `elem_start = 0` for nested arrays (strings)

does that make sense?